bottledaemon/bottledaemon.py

```diff
@@ -25,11 +25,12 @@ def __locked_pidfile(filename):
     lock.release()
 
 
-def daemon_run(host="localhost", port="8080", pidfile=None, logfile=None):
+def daemon_run(app=None, host="localhost", port="8080", pidfile=None, logfile=None):
```

Comment:
It's always best to *add* the default params to existing functions at the *end* of other params. This way, it will not break the existing code where that function is called using positional parameters.